provider-ui/components/ProviderComponent.js

```diff
@@ -256,7 +274,7 @@ class ProviderComponent extends React.Component {
                       onClick={self.handleToggle()}
                     >
                       <ArrowDropDownIcon />
-                    </Button>
+                    </Button> */}

```

Comment:
The CI error is because you commented/deleted the button that we're using for selecting a Provider. You may want to move this `data-cy="select_provider"` property to your new button.

Reply:
Tried it, still facing the same error 😶

Reply:
The error is different now :)

The previous error was:

```bash
AssertionError: Timed out retrying after 4000ms: Expected to find element: `[data-cy="select_provider"]`, but never found it.
```

Now, the problem is that, by default, the **Meshery** provider is auto-selected. But, if you want to change the Provider (after selecting one or if it is already selected by default in this test), you're not able to change it anymore.

I think we have two options:
1. We should allow changing the Provider after selecting one or;
2. Send the request immediately after selecting one option in the Dropdown (and not require to click again in the button to send the request)

@leecalcote what do you think about these two options?

Reply:
Fixed this 🎉